spec/factories/financial_transaction_type.rb

```diff
@@ -0,0 +1,13 @@
+require 'factory_girl'
+
+FactoryGirl.define do
+
+  factory :financial_transaction_class do
+    sequence(:name) { |n| Faker::Lorem.characters(rand(2..12)) + " ##{n}" }
+  end
+
+  factory :financial_transaction_type do
+    sequence(:name) { |n| Faker::Lorem.words(rand(2..4)).join(' ') + " ##{n}" }
```

Comment:
Since a type isn't valid without a class, I'd add the line `financial_transaction_class` to it. See [FactoryGirl#associations](http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Associations).